### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ux-han**
- Update Jan's new positioning
- Change banner with Jan's official logo & new positioning

### CODE REVIEW

**Comment by dan-jan** [comment]
@imtuyethan In future, please try to have a slightly more descriptive PR name (e.g. "update-readme")

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Jan - Run your own AI
+# Jan - Personal AI
 
-![](./docs/static/img/github-readme-banner.png)
+![github banner landscape](https://github.com/janhq/jan/assets/89722390/9d7b24dc-0b6f-4de2-a0a7-68d35ec1a672)
```
Comment (dan-jan) [inline README.md:3]: Given that our tagline is a lot shorter, we can probably put them side-by side. 

i.e. "Jan" Personal AI
image

File: README.md [hunk 2]
```diff
@@ -18,7 +18,7 @@
 
 > ⚠️ **Jan is currently in Development**: Expect breaking changes and bugs!
 
-Jan runs Large Language Models and AIs on your own Windows, Mac or Linux computer. Jan can be run as a desktop app, or as a cloud-native deployment.
+Jan is a powerful Personal AI built to run locally on your machine, with a rich app and plugin ecosystem.
 
 Jan is free and open source, under the GPLv3 license.
```
Comment (dan-jan) [inline README.md:24]: image

**Review by dan-jan — CHANGES_REQUESTED** [review]

**Comment by imtuyethan** [comment]
> @imtuyethan In future, please try to have a slightly more descriptive PR name (e.g. "update-readme")

My bad, will improve next time!!

**Review by dan-jan — APPROVED** [review]

**Pull request merged**